sklearn/manifold/_mds.py

```diff
@@ -318,30 +389,35 @@ class MDS(BaseEstimator):
 
     metric : bool, default=True
         If ``True``, perform metric MDS; otherwise, perform nonmetric MDS.
+        If ``solver=='svd'``, metric must be set to True.
 
-    n_init : int, default=4
+    n_init : int, optional, default=4
         Number of times the SMACOF algorithm will be run with different
         initializations. The final results will be the best output of the runs,
         determined by the run with the smallest final stress.
+        Ignored if ``solver=='svd'``.
```

Comment:
```suggestion
        Used only when ``solver=='smacof'``.
```

API wise there are so many ignored parameters for the new solver. Nothing we can do here, it's a heavy debated issue in scikit-learn for solver specific parameters in estimators.